cypress/support/generateRegistrationInput.js

```diff
@@ -0,0 +1,21 @@
+const { faker } = require('@faker-js/faker');
+
+function generateRegistrationInput() {
+  const firstName = 'A' + faker.person.firstName();
+  const lastName = 'A' + faker.person.lastName();
+  const email = faker.internet.email();
+  const number = '0981324564';
+  const address = faker.location.streetAddress();
+  const subject = faker.lorem.sentences().slice(2, 11);
```

Comment:
The `subject` field is generated using `faker.lorem.sentences().slice(2, 11)`, which might not produce a valid subject input for the form. Consider using a more appropriate method to generate a single subject name or keyword, such as `faker.lorem.word()` or a predefined list of subjects.